.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build & test with Gradle
+
+env:
+  nexusUsername: ${{ secrets.SONATYPE_USER }}
+  nexusPassword: ${{ secrets.SONATYPE_PASSWORD }}
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build & test with Gradle
+        run: ./gradlew -PnexusUsername=nexusUsername -PnexusPassword=nexusPassword build
```

Comment:
it looks like there is should be run of tests, but I don't see it

Reply:
`gradle build` command implies tests run.